python/instrumentation/openinference-instrumentation-guardrails/tests/openinference/instrumentation/guardrails/test_instrumentor.py

```diff
@@ -0,0 +1,65 @@
+import pytest
+from guardrails import Guard
+from guardrails.validators import TwoWords
+from guardrails.utils.llm_response import LLMResponse
+from unittest.mock import patch
+from openinference.instrumentation.guardrails import GuardrailsInstrumentor
+from opentelemetry import trace as trace_api
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import SimpleSpanProcessor
+from opentelemetry.sdk.trace.export.in_memory_span_exporter import InMemorySpanExporter
+from opentelemetry.sdk.resources import Resource
+
+@pytest.fixture()
+def in_memory_span_exporter() -> InMemorySpanExporter:
+    return InMemorySpanExporter()
+
+@pytest.fixture()
+def tracer_provider(in_memory_span_exporter: InMemorySpanExporter) -> TracerProvider:
+    resource = Resource(attributes={})
+    tracer_provider = TracerProvider(resource=resource)
+    tracer_provider.add_span_processor(SimpleSpanProcessor(in_memory_span_exporter))
+    return tracer_provider
+
+@pytest.fixture(autouse=True)
+def setup_guardrails_instrumentation(tracer_provider: TracerProvider) -> None:
+    GuardrailsInstrumentor().instrument(tracer_provider=tracer_provider)
+    yield
+    GuardrailsInstrumentor().uninstrument()
```

Comment:
maybe assert that the functions are un-wrapt

Reply:
if we do that, maybe put it in the uninstrument method?